mergin/client.py

```diff
@@ -618,3 +626,67 @@ def get_projects_by_names(self, projects):
 
         resp = self.post("/v1/project/by_names", {"projects": projects}, {"Content-Type": "application/json"})
         return json.load(resp)
+
+    def download_file(self, project_dir, file_path, output_filename, version=None):
+        """
+        Download project file at specified version. Get the latest if no version specified.
+
+        :param project_dir: project local directory
+        :type project_dir: String
+        :param file_path: relative path of file to download in the project directory
+        :type file_path: String
+        :param output_filename: full destination path for saving the downloaded file
+        :type output_filename: String
+        :param version: optional version tag for downloaded file
+        :type version: String
+        """
+        try:
+            mp = MerginProject(project_dir)
+        except InvalidProject as e:
+            err = f"Couldn't create Mergin project for {project_dir}: {repr(e)}"
+            self.log.error(err)
+            raise ClientError(err)
+
+        project_path = mp.metadata["name"]
+        ver_info = f"at version {version}" if version is not None else "at latest version"
+        self.log.info(f"Getting {file_path} from {project_path} {ver_info}")
+        job = download_file_async(self, project_path, file_path, output_filename, version=version)
+        if job is None:
+            return
```

Comment:
is it possible that a 'none' job is returned? it seems to me it always ither returns a job or raises an exception